app/scripts/controllers/person/tabSubjects.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+angular.module('admissionSystemApp')
+  .controller('tabSubjects', ['$scope', '$modal', '$rootScope', 'DictionariesSvc', function ($scope, $modal, $rootScope, DictionariesSvc) {
+
+
+    // temp data --------begin---------
+    $scope.entirePerson.enrolmentsubjects = [
+      {
+        "id": 1,
+        "personId": 11,
+        "personPaperId": 3,
+        "enrolmentSubjectId": 3,
+        "mark": 3
+      },
+      {
+        "id": 2,
+        "personId": 11,
+        "personPaperId": 2,
+        "enrolmentSubjectId": 2,
+        "mark": 5
+      }
+    ];
+    // temp data --------end---------
+
+    $scope.getSubjectTitleById = function (id) {
+      var subjectTitle = '';
+      angular.forEach($scope.subjects, function (subject) {
+        if (subject.id === id) {
+          subjectTitle = subject.name;
+        }
+      });
+      return subjectTitle
+    };
+
+    $scope.filterExistsSubjects = function (subject) {
+      return !_.find($scope.entirePerson.enrolmentsubjects, function (item) {
+        return item.enrolmentSubjectId == subject.id
+      });
+    };
+
+    $scope.enrolSubjIds = DictionariesSvc.getEnrolmentsSubjects().then(function (subjs) {
+      $scope.subjects = subjs;
+    });
+
+    $scope.addPersonSubject = function () {
+      var newSubj = {};
```

Comment:
You don't have to create new variable there, just send object in push function, like
```
.push({
  enrolmentSubjectId: $scope.enrolmentsubject.id
  mark: $scope.subjectMark
})
```